src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Legacy/HtmlTokenizer.cs

```diff
@@ -98,20 +111,22 @@ protected override string GetTokenContent(SyntaxKind type)
                 case SyntaxKind.SingleQuote:
                     return "'";
                 case SyntaxKind.Whitespace:
-                    if (Buffer[0] == ' ')
-                    {
-                        return " ";
-                    }
-                    if (Buffer[0] == '\t')
+                    switch (Buffer[0])
```

Comment:
nit: would a switch expression (rather than statement) be more concise here?

Reply:
I considered that, but it requires adding another call to `base.GetTokenContent(type)` to avoid changing the current fall-through behavior. So, I opted for a regular switch to let the fall-through continue to work as it did before. So, it'd realistically only save 2 lines or so.